### DISCUSSION TITLE
问问大家，如果要更换岛的域名是不是用这个插件，怎么个用法捏

### DISCUSSION THREAD

**Discussion by cng10001** [jxxghp/MoviePilot#1214] [问题咨询]
![image](https://github.com/jxxghp/MoviePilot/assets/115156545/10aecb91-b580-4d99-8c14-e33c18dbc354)
![image](https://github.com/jxxghp/MoviePilot/assets/115156545/e22c795f-4bd8-49d4-ac6c-5cba5157dd6e)
配置Json的base64编码(utf-8)是什么

**Comment by ddgocl** [comment]
请问岛添加成功了吗？还有M头

**Reply by ddgocl** [reply]
ptchdbits.co
岛我用这个域名登录成功了

**Reply by cng10001** [reply]
馒头是https://xp.m-team.io
